srcpkgs/ananicy/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'ananicy'
+pkgname=ananicy
+version=2.2.1
+revision=1
+wrksrc=Ananicy-${version}
+depends="bash schedtool python3"
+short_desc="Ananicy daemon"
+maintainer="linuxer <linuxer@disroot.org>"
+license="GPL-3.0-only"
+homepage="https://github.com/Nefelim4ag/Ananicy"
+distfiles="https://github.com/Nefelim4ag/Ananicy/archive/refs/tags/${version}.tar.gz"
+checksum=3e33c3c84d29f93b64079e67eb56c39a4d40b0591e52db171156981b72f648b1
+
+do_install() {
+	make install PREFIX="${DESTDIR}"
+}
```

Comment:
It would be nice to upstream a fix to support DESTDIR.

Reply:
Thanks for your suggestion ;)